docs/how-to/customising_news_output.rst

```diff
@@ -0,0 +1,80 @@
+.. _customising_news_output:
+
+#############################
+Customising news output
+#############################
+
+By default, a News & Blog page will simply list items, using the ``article_list.html`` template,
```

Comment:
We seem to be branding this product as: Aldryn News & Blog, News and Blog, News & Blog. We should probably be consistent throughout.